src/App.tsx

```diff
@@ -1,8 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { getPreparedTodos } from './api';
+import { TodoList } from './components/TodoList/TodoList';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+export const App: React.FC = () => {
+  const [isLoaded, setLoad] = useState<boolean>(false);
+  const [isLoading, setLoading] = useState<boolean>(false);
```

Comment:
примитивы не указываются
```suggestion
  const [isLoaded, setLoad] = useState(false);
  const [isLoading, setLoading] = useState(false);
```